### PR TITLE
TTAndroid - Accessibility hint for away status

### PR DESCRIPTION
**Pull request by Oreonan**

### CODE REVIEW

**Comment by Oreonan** [comment]
Hmmmm, I made a mistake, now if an user is away, we only have "away" announce in his nickname, we don't have his nickname announce anymore. @bear101 Do you think moving the away status check before other conditions can correct the problem?

File: Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java [hunk 1]
```diff
@@ -1138,9 +1138,8 @@ else if(item instanceof User) {
                 if(user.nUserID == ttservice.getTTInstance().getMyUserID()) {
                     talking = ttservice.isVoiceTransmitting();
                 }
-                
+                String name1 = Utils.getDisplayName(getBaseContext(), user);
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java:1141]: Why do we extract "name1" when we have "name" already?

File: Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java [hunk 2]
```diff
@@ -1158,6 +1157,9 @@ else if(item instanceof User) {
                         icon_resource = away? R.drawable.man_orange : R.drawable.man_blue;
                     }
                 }
+                if(away) {
+                    nickname.setContentDescription(getString(R.string.user_state_away, name1));
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java:1161]: Shouldn't it be "status" instead of "nickname"?

**Comment by Oreonan** [comment]
Ah, I didn't know it worked too. I will do that

**Pull request merged**